### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [Leaflet/Leaflet#8023] [Q&A]
Hello!
I have just found about Leaflet.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to make System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application. 

Can I make SRS document for this project?

**Comment by Falke-Design** [comment]
We already have a Browser support list on https://leafletjs.com/. What else would be in the SRS document then the Browser support?

![grafik](https://user-images.githubusercontent.com/19800037/155605646-800ffcc2-bfb6-49b7-8dda-20961c5c3d58.png)

**Reply by TemirlanAidarov** [reply]
I can write the following things in one document
1. Introduction
           1.1 Purpose
           1.2 Intended Audience
           1.3 Intended Use
           1.4 Scope
           1.5 Definitions and Acronyms

2. Overall Description
           2.1 User Needs
           2.2 Assumptions and Dependencies

3. System Features and Requirements
            3.1 Functional Requirements
            3.2 External Interface Requirements
            3.3 System Features
            3.4 Nonfunctional Requirements

And make a tutorials for this project

**Reply by Falke-Design** [reply]
You make this document but I don't really see the benefit because there no system requirements. Maybe it is clearer when you have made it. But it can't guarantee you that we will merge it, it will go through the same review process as every Pull Request.
